lib/engine/tile.rb

```diff
@@ -222,6 +223,21 @@ def preferred_city_town_edges
       @preferred_city_town_edges ||= compute_city_town_edges
     end
 
+    def reserved_by?(corporation)
```

Comment:
do we need to store reservations or can we just get them from the cities?

Reply:
No, we need to store since they're disconnected from the cities